incubator/mariadb-cluster/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: mariadb-cluster
-version: 0.5.2
```

Comment:
as you did major changes in this chart you should bump the major version: 
`version: 1.0.0`